common-messaging/src/main/scala/org/genivi/sota/messaging/Messages.scala

```diff
@@ -112,6 +112,14 @@ object Messages {
                                     pkgSize: Long, pkgChecksum: String,
                                     timestamp: Instant = Instant.now()) extends BusMessage
 
+  final case class DeltaRequest(id: Uuid, namespace: Namespace, from: String, to: String,
+                                timestamp: Instant = Instant.now) extends BusMessage
+
+  final case class GeneratedDelta(id: Uuid, namespace: Namespace, from: String, to: String, summary: OstreeSummary,
+                                  uri: Uri, size: Long) extends BusMessage
```

Comment:
Will the message parse if I don't have summary in the definition of `GeneratedDelta`?

Yeah it's the campaign id